eks_fargate/datadog_checks/eks_fargate/eks_fargate.py

```diff
@@ -0,0 +1,33 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.base import AgentCheck
+
+
+class EksFargateCheck(AgentCheck):
+    """
+    Generate a heartbeat for Amazon EKS on AWS Fargate workloads
+    """
+
+    def __init__(self, name, init_config, instances):
+        super(EksFargateCheck, self).__init__(
+            name, init_config, instances,
+        )
+        pod_name = os.getenv("HOSTNAME")
+        virtual_node = os.getenv("DD_KUBERNETES_KUBELET_NODENAME")
```

Comment:
If unset `'fargate' in virtual_node` will fail

```suggestion
        virtual_node = os.getenv("DD_KUBERNETES_KUBELET_NODENAME", "")
```

Reply:
it will live in a pod so it should be immutable. But ok, I'll change

Reply:
No I mean that becomes `'fargate' in None`